posts/2021/weather-station-data-preprocess.ipynb

```diff
@@ -0,0 +1,1934 @@
+{
```

Comment:
<blockquote>check out the Pythia Foundations material on this</blockquote><p> </p><p>Maybe add a link to the exact page with netCDF content?</p><p> </p><blockquote>but we do not helpful time information</blockquote><p> </p><p>we don not have ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>